lib/friendly_shipping/services/ship_engine/serialize_label_shipment.rb

```diff
@@ -45,8 +45,19 @@ def serialize_packages(packages)
                 unit: "ounce"
               }
             }
-            if package.container.properties[:usps_package_code]
-              package_hash.merge!(package_code: package.container.properties[:usps_package_code])
+
+            package_code = package.container.properties[:usps_package_code] || "package"
+            package_hash.merge!(package_code: package_code)
+
+            if package_code == 'package'
```

Comment:
@mamhoff Could `: usps_package_code` actually have the value "package" or is that just a default value?